### PR TITLE
fix(order): price without tax addition in order step1

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
MFRWW-454

### Requirements

* The step1 of order should show the price both with and without tax.
* The price format for Germany should be of the format: 1.11 € (1.11 € netto)

### Description of the Change

* The step1 of order is modified to show the price both with and without tax.
* The price format for Germany is changed to the format: 1.11 € (1.11 € netto)

*Depends on https://github.com/ovh-ux/ovh-module-exchange/pull/121*

### CODE REVIEW

File: src/exchange/account/order/account-order.controller.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ angular
                 exchangeServiceInfrastructure
             };
 
+            User.getUser()
+                .then((currentUser) => {
+                    this.isGermanSubsidiary = currentUser.ovhSubsidiary === "DE";
+                });
```
Comment (frenautvh) [inline src/exchange/account/order/account-order.controller.js]: .catch ?

**Comment by frenautvh** [comment]
why do you handle two separate trads? it looks like it could simply be translated differently in DE language

File: src/exchange/account/order/account-order.controller.js (hunk 1 above)
Comment (ganeshkumar1989) [reply src/exchange/account/order/account-order.controller.js]: The catch block is added :)

**Comment by ganeshkumar1989** [comment]
For Germany, the price format is entirely different. Hence, if we have a format for France, it cannot be translated to German as such. Hence we need a new translation id, just for Germany price format

File: src/exchange/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -434,6 +434,8 @@
    <translation id="exchange_ACTION_order_accounts_step1_pay_type" qtlid="139325">Type de paiement :</translation>
    <translation id="exchange_ACTION_order_accounts_step1_pay_type_01" qtlid="79530">mensuel</translation>
    <translation id="exchange_ACTION_order_accounts_step1_pay_type_12" qtlid="79542">annuel</translation>
+   <translation id="exchange_ACTION_order_accounts_step1_price">{0} HT ({1} TTC)</translation>
+   <translation id="exchange_ACTION_order_accounts_step1_price_germany">{0} ({0} TTC)</translation>
```
[the remark this answers is not in the document]
Comment (ganeshkumar1989) [reply src/exchange/translations/Messages_fr_FR.xml]: This is fixed. Thanks :)!

**Comment by ovh-cds** [comment]
CDS Report build#479.0 ✘  
* build
  * build ✔ 
  * test ✘

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: src/exchange/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -434,6 +434,8 @@
    <translation id="exchange_ACTION_order_accounts_step1_pay_type" qtlid="139325">Type de paiement :</translation>
    <translation id="exchange_ACTION_order_accounts_step1_pay_type_01" qtlid="79530">mensuel</translation>
    <translation id="exchange_ACTION_order_accounts_step1_pay_type_12" qtlid="79542">annuel</translation>
+   <translation id="exchange_ACTION_order_accounts_step1_price">{0} HT ({1} TTC)</translation>
+   <translation id="exchange_ACTION_order_accounts_step1_price_with_tax_only">{0} ({0} TTC)</translation>
```
Comment (FredericEspiau) [inline src/exchange/translations/Messages_fr_FR.xml]: `{0} TTC` should be enough 😃

Comment (ganeshkumar1989) [reply src/exchange/translations/Messages_fr_FR.xml]: This is changed

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by ovh-cds** [comment]
CDS Report build#569.0 ✘  
* build
  * build ✔ 
  * test ✘

**Pull request merged**